### ISSUE TITLE
Accounts are not sorted in order

### ISSUE THREAD

**Issue by InfiniteQE** [cryptoadvance/specter-desktop#1118]
When I add a Device and import multiple accounts, Specter does not correctly sort by numerical order as per the Purpose column.

ie: with 20 accounts imported (80 keys), Specter displays them in the order

```
#9
#8
#7
#6
#5
#4
#3
#2
#19
#18
#17
#16
#15
#14
#13
#12
#11
#10
#1
#0
```

**Comment by k9ert** [comment]
I'm not sure what you mean. New Accounts usually get added at the bottom like e.g. in the screenshot where i added account 8 first and then 7. 
![image](https://user-images.githubusercontent.com/117085/116568934-3857a580-a909-11eb-9f10-73b5f3969b39.png)
Or do you talk about a different import?

**Comment by moneymanolis** [comment]
I get this now. It looks like the ordering logic is focusing on the first digit only. Will check it.

**Comment by moneymanolis** [comment]
Fixed with #1195 and the corresponding bug fix #1204.
